cms/djangoapps/contentstore/toggles.py

```diff
@@ -38,9 +38,32 @@
     module_name=__name__
 )
 
+# Waffle flag to enable olx validation during course import.
+# .. toggle_name: course_import_olx_validation
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Studio Import
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2021-04-01
+# .. toggle_target_removal_date: None
```

Comment:
Please set a target removal date. It's fine to have a WaffleFlag for the transition, but we don't want this as a permanently supported permutation.